changelog.md

```diff
@@ -10,6 +10,14 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.4.1 July 8, 2020
+* Fixed the Search and Replace plugin incorrectly including zero-width caret characters in search results.
+* Fixed dragging and dropping unsupported files navigating the browser away from the editor.
+* Fixed undo levels not created on browser handled drop or paste events.
+* Fixed content in an iframe element parsing as dom elements instead of text content.
```

Comment:
s/dom/DOM/ ?

Reply:
Can't be changed here. You were the one that made these changes in the TinyMCE repo, so it's just a copy of that.